solidity/contracts/deposit/Deposit.sol

```diff
@@ -109,9 +109,9 @@ contract Deposit is DepositFactoryAuthority {
         address _TBTCDepositToken,
         address _FeeRebateToken,
         address _VendingMachine,
-        uint256 _m,
-        uint256 _n,
-        uint256 _lotSizeSatoshis
+        uint16 _m,
+        uint16 _n,
```

Comment:
We do need to make sure either keep-ecdsa is updated to this, or we consider not making the change here. We may not be willing to make this change in keep-ecdsa right now.

Reply:
I have a note in the PR description. We don't need to update `keep-ecdsa`.
`TBTCSystem.requestNewKeep` calls out to `_keepFactory.openKeep`, but `requestNewKeep` still takes `uint256`.
`function requestNewKeep(uint256 _m, uint256 _n, uint256 _bond)`
`uint16` params are passed into it from `depositFunding`, but they should get implicitly casted.

Reply:
Sorry, I'm not worried that we'll pass bad data to Keep, but rather that it feels like we should reflect Keep's capabilities in this interface. If Keep uses uint256, we should probably expose the same; if Keep uses uint16, we should do that instead. I think having Keep use uint16 makes sense, the top limit there feels reasonable… But we should discuss.

Reply:
Not having divergent assumptions makes sense. The change saves about 2k gas

Reply:
We still need to work through this on the Keep side; for now, we're going to keep what's here, here. Even if Keep supports larger groups, I think it's survivable for tBTC has a smaller limitation.